.github/workflows/ci.yml

```diff
@@ -0,0 +1,102 @@
+---
+name: CI
+on:
+  pull_request:
+  push:
```

Comment:
The only downside to this is that this doubles the CI runs for contributors. Since this matches Both the PR and the Push.

Maybe we can figure out a filter so this doesn't happen.

Reply:
I copied this from the CI job for the prometheus repo https://github.com/prometheus/prometheus/blob/main/.github/workflows/ci.yml#L5. Does that job run twice? If this runs twice here when someone opens a PR, I'm happy to filter to the master/main branches.